public/scss/pages/Placeorder.module.scss

```
@@ -32,51 +32,113 @@
 {
   position: relative;
   margin-bottom: 12px;
+
+  // NOTE: IF applied to street will effect country & address detail label, but I can't find a way to apply this to country only
+  // label[for="street"]
+  // {
+  //   margin-left: 42px;
+  // }
+
+  input[id="postalCode"]
+  {
+    @include fixedWidth(45%);
+  }
   
   label
   {
-    @include position(absolute, 8px, auto, auto, 16px);
+    @include position(absolute, 8px, auto, auto, 0);
     @include SmallSmall;
     color: $color_text_secondary;
     z-index: 2;
+    margin-left: 16px;
 
     & > span
     {
       color: $color_red_other;
     }
   }
 
-  [class^=" css-"]
+  &[class$="-placeholder"],
+  &[class$="-Placeholder"],
+  &[class$="-6us33n"]
   {
-    &[class$="-placeholder"],
-    &[class$="-Placeholder"],
-    &[class$="-6us33n"]
-    {
-      @include SmallSmall;
-      white-space: nowrap;
-      word-break: keep-all;
-    }
+    @include RegulerReguler;
+    white-space: nowrap;
+    word-break: keep-all;
+  }
 
-    &[class$="-SingleValue"] 
-    {
-      padding-top: 18px;
-      white-space: nowrap;
-      word-break: keep-all;
-      text-overflow: ellipsis;
-    }
+  &[class$="-SingleValue"] 
+  {
+    padding-top: 18px;
+    white-space: nowrap;
+    word-break: keep-all;
+    text-overflow: ellipsis;
```

sepertinya bisa diganti mixin
```suggestion
    @include textOverflow();
```